test/standalone/webpack.config.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+const path = require('path');
+
+const common = {
+  context: path.resolve(__dirname, '..', '..'),
+  mode: 'development',
+  entry: path.resolve(__dirname, 'main.js'),
+  target: 'web',
+  output: {
+    filename: 'client.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  resolve: {
+    extensions: ['.ts', '.tsx', '.js', '.jsx', '.mjs'],
+  },
+  module: {
+    // strictExportPresence: true,
+    rules: [
+      {
+        test: /\.(png|jpg|gif)$/i,
+        use: [
+          {
+            loader: 'url-loader',
+            options: {
+              // limit: false,
+            },
+          },
+        ],
+      },
+      {
+        test: /jsx?$/,
+        // type: "javascript/auto",
+        exclude: [/node_modules/],
+        use: [
```

Comment:
can we defer to our standard babel rc file for this?

Reply:
Yes, cleaned up